include/libserver/data/DataDefinitions.hpp

```diff
@@ -252,6 +238,7 @@ struct Character
   dao::Field<std::vector<Uid>> eggs{};
 
   dao::Field<std::vector<Uid>> housing{};
+  dao::Field<uint32_t> incubatorSlots{};
```

Comment:
Shouldn't this be defined by the housing itself? If user has in its housing a double incubator they surely must have two slots. Let it be implied rather than separating it to a different field, that is error prone. I'm pretty sure that's also one of the laws for defining good databases, which in this case, these data objects are pretty much internal databases.